optimize/backend/src/test/java/io/camunda/optimize/service/util/BpmnModelUtilTest.java

```diff
@@ -0,0 +1,106 @@
+import static io.camunda.optimize.util.ZeebeBpmnModels.START_EVENT;
+import static io.camunda.optimize.util.ZeebeBpmnModels.USER_TASK;
+import static io.camunda.optimize.util.ZeebeBpmnModels.createSimpleServiceTaskProcess;
+import static io.camunda.optimize.util.ZeebeBpmnModels.createSimpleUserTaskProcess;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import io.camunda.optimize.dto.optimize.FlowNodeDataDto;
+import io.camunda.zeebe.model.bpmn.Bpmn;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.Set;
+import org.camunda.bpm.model.bpmn.BpmnModelInstance;
+import org.junit.jupiter.api.Test;
+
+public class BpmnModelUtilTest {
+
+  public static final String CUSTOMER_ONBOARDING = "CustomerOnboarding";
+  public static final String END_EVENT_TYPE = "endEvent";
+  public static final String SERVICE_TASK_TYPE = "serviceTask";
+
+  public static final String SIMPLE_SERVICE_TASK_PROCESS =
+      Bpmn.convertToString(createSimpleServiceTaskProcess(CUSTOMER_ONBOARDING));
+
+  @Test
+  void shouldParseBpmnModel() {
+    final BpmnModelInstance modelInstance =
+        BpmnModelUtil.parseBpmnModel(SIMPLE_SERVICE_TASK_PROCESS);
+    assertThat(modelInstance).isNotNull();
+  }
+
+  @Test
+  void shouldExtractFlowNodeData() {
+    final List<FlowNodeDataDto> flowNodeData =
+        BpmnModelUtil.extractFlowNodeData(SIMPLE_SERVICE_TASK_PROCESS);
+    assertThat(flowNodeData).isNotNull();
+    assertThat(flowNodeData.isEmpty()).isFalse();
+    FlowNodeDataDto endEventflowNodeDataDto = flowNodeData.get(0);
+    assertThat(endEventflowNodeDataDto.getId()).isNotNull();
+    assertThat(endEventflowNodeDataDto.getId()).isEqualTo(END_EVENT);
+    assertThat(endEventflowNodeDataDto.getType()).isEqualTo(END_EVENT_TYPE);
+
+    FlowNodeDataDto serviceTaskflowNodeDataDto = flowNodeData.get(1);
+    assertThat(serviceTaskflowNodeDataDto.getId()).isNotNull();
+    assertThat(serviceTaskflowNodeDataDto.getId()).isEqualTo(SERVICE_TASK);
+    assertThat(serviceTaskflowNodeDataDto.getName()).isEqualTo(SERVICE_TASK);
+    assertThat(serviceTaskflowNodeDataDto.getType()).isEqualTo(SERVICE_TASK_TYPE);
+
+    FlowNodeDataDto startEventflowNodeDataDto = flowNodeData.get(2);
+    assertThat(startEventflowNodeDataDto.getId()).isNotNull();
+    assertThat(startEventflowNodeDataDto.getId()).isEqualTo(START_EVENT);
+    assertThat(startEventflowNodeDataDto.getName()).isEqualTo(START_EVENT);
+    assertThat(startEventflowNodeDataDto.getType()).isEqualTo(START_EVENT);
+  }
+
+  @Test
+  void shouldExtractUserTaskNames() {
+    final String bpmnModelInstance =
+        Bpmn.convertToString(createSimpleUserTaskProcess(CUSTOMER_ONBOARDING));
+    final Map<String, String> userTaskNames = BpmnModelUtil.extractUserTaskNames(bpmnModelInstance);
+    assertThat(userTaskNames).isNotNull();
+    assertThat(userTaskNames.isEmpty()).isFalse();
+
+    assertThat(userTaskNames.containsKey(USER_TASK)).isTrue();
```

Comment:
and this could be `assertThat(userTaskNames).isNotNull().isNotEmpty().containsKey(USER_TASK);`

Please apply the same changes to all tests in this file

Reply:
fixed